docs/05-module-block-5/02-learning-goals.adoc

```diff
@@ -14,54 +14,34 @@ Zudem vermittelt er Strategien, um mehrere Modelle, und die daraus erwachsenden
 
 [[LZ-5-2]]
 ==== LZ 5-2: Vor- und Nachteile eines Team-übergreifenden Modells abwägen können
-* Die TN kennen Conway's Law footnote:7[Cf.: <<conway>>] und können es auf die (Kommunikations-)Struktur von Entwicklungs-Teams und Software-Artefakten anwenden.
+* Die TN kennen Conway's Law footnote:7[Cf.: <<conway>>] und seine Auswirkungen auf Software-Architektur.
```

Comment:
Könntet ihr die Fußnoten (ihr habt nur eine Handvoll) einfach in Klammern hinter den Text schreiben? Fußnoten werden aktuell immer am Kapitelende gerendert, und das noch dazu auf einer neuen Seite. Das sieht bisschen doof aus, und bläht das Dokument unnötig auf. Wäre das ok für euch?